### PR TITLE
Rehydrate configuration files and mount them

### PR DESCRIPTION
**Pull request by brandomr**

### CODE REVIEW

**Comment by marshHawk4** [comment]
To test this you will need to:
1. update the settings.py file - I still updated the config.ini but might not be used anymore
2. You can load in most of the information you will need by navigating to the examples pythia folder and running python3 Examples.py. This will load the model, the directives, the outputfiles indexes. I couldn't get the config index to load this way unfortunately. 
3. Load the config through the dojo endpoint by copying the config_pythia.json data into this endpoint: http://localhost:8000/#/Dojo/create_configs_dojo_config_post
4. You should now have the model pretty much set up. You shouldn't have to make a model_configs folder in the dmc directory because the code should make that for you. If it fails the permissions might not be working correctly. 
5. Run the model by copying the TEST_RUN.json data from in the pythia example dir into the dojo endpoint: http://localhost:8000/#/Runs/create_run_runs_post.
6. Check to see if a pythia_my_test_run_1 folder is in the model_configs dir and check if et_docker.json is in that. At this step the file should be hydrated with the parameters you set in the TEST_RUN.json data
7. Also look to see if there are any pp.csv file in the results directory. If that is there then you are basically where I am at.

**Comment by brandomr** [comment]
@marshHawk4 were you able to get the model output normalized and pushed to S3?

**Comment by marshHawk4** [comment]
To test the dummy-model. Run the 
1. update the settings.py file for dojo- I still updated the config.ini but might not be used anymore
2. For the dummy-model can load in most of the information by navigating to the examples dummy-model folder and running python3 Examples.py. This will load the model, the directives, the outputfiles, and config indexes. However I got an error loading the config index this way.
3. If you need to manually load the configs through the dojo endpoint. Then copy the config_dummy_model.json data into this endpoint: http://localhost:8000/#/Dojo/create_configs_dojo_config_post
4.Run the model by copying the TEST_RUN.json data from in the dummy_model example dir into the dojo endpoint: http://localhost:8000/#/Runs/create_run_runs_post.
5. Check to see if a my_dummy_test_* folder is in the model_configs dir and check if parameters.json and fakeDataType.json are in there and hydrated. At this step the file should be hydrated with the parameters you set in the TEST_RUN.json data
Also after the run completes look to see if there is an output file in the results directory. If that is there then you are basically where I am at.
6. Mixmasta didn't work, but this is probably because I don't have the latest fixes on the version I am using. Still getting string errors.

File: api/.env [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
 ELASTICSEARCH_PORT=9200
-DMC_URL=SET_DMC_URL_HERE
+DMC_URL=192.168.1.155
 DMC_PORT=8080
 DMC_USER=jataware
 DMC_PASSWORD=wileyhippo
-DOJO_URL=http://SET_DOJO_URL_HERE:8000
\ No newline at end of file
+DOJO_URL=http://192.168.1.155:8000
+dmc_local_dir=/home/kyle/code/dojo/dmc
```
Comment (brandomr) [inline api/.env]: @marshHawk4 this should be consistently cased as the other variables.

File: api/config.ini [hunk 2]
```diff
@@ -3,10 +3,11 @@ URL = elasticsearch
 PORT = 9200
```
Comment (brandomr) [inline api/config.ini]: This file should be removed.

File: dmc/dags/model-xform.py [hunk 3]
```diff
@@ -109,41 +175,89 @@ def RunExit(**kwargs):
...
+    run_id = kwargs['dag_run'].conf.get('run_id')
+    model_id = kwargs['dag_run'].conf.get('model_id')
+    run = requests.get(f"{dojo_url}/runs/{run_id}").json()
+
+    # TODO: this should be conditional; if the other tasks fail
+    # this should reflect the failure; job should always finish
+    run['attributes']['status'] = 'failed'
+
+    # TODO: handle additional output files
+    pth = f"https://jataware-world-modelers.s3.amazonaws.com/dmc_results/{run_id}/{run_id}_{model_id}.parquet.gzip"
```
Comment (brandomr) [inline dmc/dags/model-xform.py]: @marshHawk4 this is irrelevant; the file does not exist because something failed!

**Review by brandomr — COMMENTED** [review]
@marshHawk4 here are some general things. I will try to take care of them directly on this branch but wanted to flag them for you.

**Pull request merged**